docs/source/add_new_model.md

```diff
@@ -0,0 +1,573 @@
+# How to add a model to 🤗Transformers
```

Comment:
Nit: I think it would be best to align with other pages and use 🤗 Transformers (with a space) everywhere.